packages/manager/modules/pci/src/projects/new/new.html

```diff
@@ -0,0 +1,27 @@
+<div class="pci-projects-new row justify-content-md-center">
+    <div class="col-md-6">
```

Comment:
No `row` above

Reply:
`pci-projects-new row justify-content-md-center` on `div` above ;-)

Reply:
Didn't see it, sorry about that 😄